src/software/sensor_fusion/sensor_fusion.cpp

```diff
@@ -186,6 +187,21 @@ void SensorFusion::updateWorld(
             }
         }
         friendly_team.setUnavailableRobotCapabilities(robot_id, unavailableCapabilities);
+
+        if (robot_status_msg.has_break_beam_status() &&
+            robot_status_msg.break_beam_status().ball_in_beam())
+        {
+            friendly_robot_id_with_ball_in_dribbler = robot_id;
+            ball_in_dribbler_timeout = NUM_DROPPED_DETECTIONS_BEFORE_BALL_NOT_IN_DRIBBLER;
+        }
+        if ((!robot_status_msg.has_break_beam_status() ||
+             !robot_status_msg.break_beam_status().ball_in_beam()) &&
```

Comment:
can't this be an `else if` instead of lines 197 and 198?

Reply:
fixed